frontend/src/views/coverage/enrollment/ResidentDetails.vue

```diff
@@ -0,0 +1,385 @@
+          address1: this.address1,
+          address2: this.address2,
+          address3: this.address3,
+          city: this.city,
+          province: this.province,
+          postalCode: this.postalCode,
+          mailingAddress1: this.mailingAddress1,
+          mailingAddress2: this.mailingAddress2,
+          mailingAddress3: this.mailingAddress3,
+          mailingAddressCity: this.mailingAddressCity,
+          mailingAddressProvince: this.mailingAddressProvince,
+          mailingAddressPostalCode: this.mailingAddressPostalCode,
+          priorResidenceCode: this.priorResidenceCode,
+          otherProvinceHealthcareNumber: this.otherProvinceHealthcareNumber,
+        })
+      } catch (err) {
+        this.$store.commit('alert/setErrorAlert', `${err}`)
+      } finally {
+        this.searching = false
+      }
+    },
+    resetForm() {
+      this.groupNumber = ''
+      this.immigrationCode = ''
+      this.groupMemberNumber = ''
+      this.visaIssueDate = null
+      this.departmentNumber = ''
+      this.visaExpiryDate = null
+      this.residenceDate = null
+      this.coverageEffectiveDate = null
+      this.telephone = ''
+      this.coverageCancellationDate = null
+      this.address1 = ''
+      this.address2 = ''
+      this.address3 = ''
+      this.city = ''
+      this.province = ''
+      this.postalCode = ''
+      this.mailingAddress1 = ''
+      this.mailingAddress2 = ''
+      this.mailingAddress3 = ''
+      this.mailingAddressCity = ''
+      this.mailingAddressProvince = ''
+      this.mailingAddressPostalCode = ''
+      this.priorResidenceCode = ''
+      this.otherProvinceHealthcareNumber = ''
+      this.v$.$reset()
+      this.$store.commit('alert/dismissAlert')
+      this.searching = false
+      this.registrationOk = false
+      this.v$.$reset()
+      this.$store.commit('alert/dismissAlert')
+      this.searching = false
+    },
+  },
+  validations() {
+    return {
+      groupNumber: {
+        required,
+        validateGroupNumber: helpers.withMessage(VALIDATE_GROUP_NUMBER_MESSAGE, validateGroupNumber),
+      },
+      immigrationCode: { required },
+      groupMemberNumber: {
+        validateGroupMemberNumber: helpers.withMessage(VALIDATE_GROUP_MEMBER_NUMBER_MESSAGE, validateGroupMemberNumber),
+      },
+      visaIssueDate: { required },
+      departmentNumber: {
+        validateDepartmentNumber: helpers.withMessage(VALIDATE_DEPARTMENT_NUMBER_MESSAGE, validateDepartmentNumber),
+      },
+      visaExpiryDate: { required },
+      residenceDate: { required },
+      coverageEffectiveDate: { required },
+      telephone: {
+        validateTelephone: helpers.withMessage(VALIDATE_TELEPHONE_MESSAGE, validateTelephone),
+      },
+      coverageCancellationDate: { required },
+      address1: { required },
+      address2: {},
+      address3: {},
+      city: { required },
+      province: { required },
+      postalCode: { required },
+      mailingAddress1: {},
```

Comment:
I think these are here because the input component fails if there's not eModel? If so I should probably fix that.

Reply:
Yeah, had to include them or got undefined error related to eModel.